arango/client.py

```diff
@@ -1,5 +1,12 @@
 __all__ = ["ArangoClient"]
 
+import sys
+
+if sys.version_info < (3, 8):
+    from importlib_metadata import version
+else:
+    from importlib.metadata import version
```

Comment:
Unfortunately, importlib-metadata's import statement is not consistent, so this is required

Reply:
no longer relevant as of https://github.com/ArangoDB-Community/python-arango/pull/206/commits/4b3d067e97f715f9afa463622842bfc1ce593716